api/src/opentrons/protocol_api/labware.py

```diff
@@ -342,11 +342,16 @@ def uri(self) -> str:
     @property  # type: ignore[misc]
     @requires_version(2, 0)
     def parent(self) -> Union[str, ModuleTypes, None]:
-        """The parent of this labware.
+        """The parent of this labware---where this labware is loaded.
 
-        If the labware is on the deck, a `str` deck slot name will be returned.
-        If on a module, the parent :py:class:`ModuleContext` will be returned.
-        If off deck, `None` will be returned.
+        Returns:
+            If the labware is directly on the robot's deck, the `str` name of the deck slot.
+            On a Flex, this will be like ``"D1"``, and on an OT-2, this will be like ``"1"``.
+            See :ref:`deck-slots`.
```

Comment:
With the addition of the new page, I'd be cool with:
```suggestion
            If the labware is directly on the robot's deck, the `str` name of the :ref:`deck slot <deck-slots>`.
```

And similarly below, link rather than spelling out in docstrings.

Reply:
I think I'd prefer to keep these as-is because, at the time of writing, we're not consistent with *returned* deck slots, so it doesn't seem right to describe "general behavior" somewhere and then link to that description from everywhere.

`Protocol.loaded_modules` and `Protocol.loaded_labwares` return integers even on Flexen, but these methods return coordinate strings. Since it's currently dependent on the specific method or property you're calling, it seems appropriate to describe in the method reference.